$NetBSD: patch-lib_Sys_Filesystem_Netbsd_pm,v 1.1 2012/06/30 05:10:51 dholland Exp $

Honor PATH_FSTAB in the environment if present. Allows this package to
configure in a chroot that doesn't have an fstab file. Cloned from
the freebsd file.

--- lib/Sys/Filesystem/Netbsd.pm~	2010-07-15 19:33:42.000000000 +0000
+++ lib/Sys/Filesystem/Netbsd.pm
@@ -57,7 +57,7 @@ sub new
     my $self = bless( {}, $class );
 
     # Defaults
-    $args{fstab} ||= '/etc/fstab';
+    $args{fstab} ||= $ENV{PATH_FSTAB} || '/etc/fstab';
 
     my @mounts = qx( /sbin/mount );
     $self->readMounts( $mount_rx, [ 0, 1, 2 ], [qw(fs_spec fs_file fs_vfstype fs_mntops)], \%special_fs, @mounts );
